contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1749,8 +1751,26 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		return;
 
 	type_def = defGetTypeName(as_type);
+	type_names = type_def->names;
+	switch (list_length(type_names))
+	{
+		case 2:
+			strVal(linitial(type_names)) = get_physical_schema_name(get_cur_db_name(),strVal(linitial(type_names)));
+			break;
+		case 3:
+			linitial(type_names) = get_physical_schema_name(strVal(linitial(type_names)),strVal(lsecond(type_names)));
```

Comment:
why not strVal on left hand side

Reply:
It should be there. I have made the change.